saphir/src/redirect.rs

```diff
@@ -106,6 +108,32 @@ impl Builder {
         self
     }
 
+    #[inline]
+    pub fn cookie(mut self, cookie: Cookie<'static>) -> Builder {
+        if self.cookies.is_none() {
+            self.cookies = Some(CookieJar::new());
+        }
+
+        self.cookies.as_mut().expect("Should not happens").add(cookie);
+
+        self
+    }
+
+    #[inline]
+    pub fn cookies(mut self, cookies: CookieJar) -> Builder {
+        self.cookies = Some(cookies);
+        self
+    }
+
+    #[inline]
+    pub fn cookies_mut(&mut self) -> &mut CookieJar {
+        if self.cookies.is_none() {
+            self.cookies = Some(CookieJar::new());
+        }
+
+        self.cookies.as_mut().expect("Checked above")
```

Comment:
CookieJar implements Default right?
If only that was stabilized :cry: 
https://doc.rust-lang.org/std/option/enum.Option.html#method.get_or_insert_default
We could have simply done
```rust
    #[inline]
    pub fn cookies_mut(&mut self) -> &mut CookieJar {
        self.cookies.get_or_insert_default()
    }
```

(nothing to change, I tought I knew a cleaner way but nothing stable sadly)